src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -754,6 +754,19 @@ subroutine CanopyFluxes(bounds,  num_exposedvegp, filter_exposedvegp,
                frac_rad_abs_by_stem(p) = 0.0_r8
```

Comment:
This is not something you did. But, the comment starting in line 749 looked wrong to me, at first. And then I realized what it's saying and it is correct, just doesn't obviously correspond to the code logic.

The code logic says:

 if it is NOT a tree or shrub OR it is (tree or shrub) and dbh is less than some minimum value combine stem and leaf together, rather than calculate them separately)